src/System.Private.ServiceModel/src/System/ServiceModel/Channels/TransportDefaults.cs

```diff
@@ -152,7 +152,9 @@ public static class TransportDefaults
         public const bool RequireClientCertificate = false;
         public const int MaxFaultSize = MaxBufferSize;
         public const int MaxSecurityFaultSize = 16384;
-        public const SslProtocols SslProtocols = System.Security.Authentication.SslProtocols.Ssl3 |
+        public const SslProtocols SslProtocols = 
+                                           // SSL3 is not supported in CoreFx.
+                                           // System.Security.Authentication.SslProtocols.Ssl3 |
```

Comment:
No need to leave this line in there when there's a comment above stating it's not supported in CoreFx